src/iccpd/src/iccp_netlink.c

```diff
@@ -1123,87 +1222,411 @@ int iccp_check_if_addr_from_netlink(int family, uint8_t *addr, struct LocalInter
     return 0;
 }
 
-int iccp_sys_local_if_list_get_addr()
+#if 0
```

Comment:
can you delete dead code and not keep it around?

Reply:
Done.!